### PR TITLE
Feature/pelagos 4905 enable erddap link on dataland

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
This PR covers both PELAGOS-4901 and PELAGOS-4902, which I ended up doing on the same ticket.

### CODE REVIEW

File: config/packages/fos_elastica.yaml [hunk 1]
```diff
@@ -106,6 +106,8 @@ fos_elastica:
                                     properties:
                                         id:
                                             type: 'keyword'
+                hasErddap:
+                    type: 'text'
```
Comment (mickel1138) [inline config/packages/fos_elastica.yaml:110]: This doesn't return text.

Comment (fightingtexasaggie) [reply config/packages/fos_elastica.yaml:110]: I wasn't sure if I could index a bool like that...good to know. I'll fix.

File: templates/Dataland/v2/download-card.html.twig [hunk 2]
```diff
@@ -33,6 +33,18 @@
                     Files
                 </div>
                 <div class="card-body">
+                    <div>
+                        {% for link in dataset.datasetSubmission.datasetLinks|default %}
+                            {% if(link.name == 'ERDDAP') %}
+                                    <button type="button" class="btn btn-primary btn-sm">
+                                        <img src="{{ asset('build/images/download-icon.svg') }}">
```
Comment (fightingtexasaggie) [inline templates/Dataland/v2/download-card.html.twig:40]: if anyone has better svg-foo than me and can change this to white, please suggest. I didn't want to resort to a .png if I could avoid it.

File: templates/Dataland/v2/download-card.html.twig [hunk 3]
```diff
@@ -33,18 +33,18 @@
...
-                                    </button>
-                                </div>
-                                <hr>
-                            {% endif %}
-                        {% endfor %}
+                        {% if dataset.datasetSubmission.erddapurl|default %}
+                            <div>
+                                <a href="{{ dataset.datasetSubmission.erddapurl }}">
+                                <button type="button" class="btn btn-primary btn-sm">
+                                    <img src="{{ asset('build/images/download-icon.svg') }}">
```
Comment (mickel1138) [inline templates/Dataland/v2/download-card.html.twig:40]: Use font-awesome

**Review by praneethpr — DISMISSED** [review]
Tested and works!

**Review by mickel1138 — DISMISSED** [review]
LGTM

**Review by mickel1138 — APPROVED** [review]

**Pull request merged**